src/main/java/com/paneedah/weaponlib/EntityShellCasing.java

```diff
@@ -52,7 +52,6 @@ public EntityShellCasing(PlayerWeaponInstance weaponInstance, World world, Entit
         this.weaponInstance = weaponInstance;
     }
 
-    @Override
```

Comment:
Why did you remove the overide???

Reply:
I always report an error every time I compile, I don't know why

Reply:
What is the error?